### DISCUSSION TITLE
SAE in VM - mit RS485 Gateway?

### DISCUSSION THREAD

**Discussion by runn8** [camueller/SmartApplianceEnabler#248] [Installation und Betrieb]
Hallo!

Kann der SAE auch direkt als VM auf einem Microsoft Hyper-V Server installiert werden? Falls ja, welches Basis OS (Linux) nehme ich hierfür am besten? Ich würde dann versuchen, den RS485 zu USB Dongle direkt in die VM zu schleifen, um 2-3 SDM630 Stromzähler mittels MODBus direkt am SAE anzubinden.

Alternativ wäre auch ein RS485-Gateway denkbar, aber welches funktioniert mit SAE?

Vielen Dank für jeden Tipp!

**Comment by camueller** [comment]
Der SAE ist ja nur eine Java-Applikation. Wenn Du das beachtest, solltest Du es zum Laufen bekommen können.
Allerdings solltest Du keinen Support (von mir) erwarten.

**Reply by runn8** [reply]
OK, danke. Das sollte ich schaffen.
Einzig meine zweite Frage wäre noch offen: Kann der SAE auch mit RS485-Gateways umgehen? Hat das schon einmal jemand erfolgreich geprüft?

Danke sehr

**Reply by premultiply** [reply]
Wenn sie dumm sind tunneln sie nur ModbusRTU über TCP, bessere Geräte können auch optional eine Übersetzung ModbusRTU <-> ModbusTCP.
Dann kommt es drauf an ob der Master (hier SAE) auch jeweils beide Protokolle bzw. "das Richtige" über TCP spricht.
